app/src/components/attachments/FileUploadItem.tsx

```diff
@@ -210,6 +210,19 @@ const FileUploadItem: React.FC<IFileUploadItemProps> = (props) => {
         return <li key={index}>{item}</li>;
       }
 
+      if (item['treatmentUnitId']) {
+        return (
+          <li key={index}>
+            TU_ID:{item['treatmentUnitId']}
+            <ul>
+              {item['errors'].map((itemError: string) => {
+                return <li key={itemError}>{itemError}</li>;
+              })}
+            </ul>
+          </li>
+        );
+      }
```

Comment:
This really shouldnt be in this component, as its not specific to treatments.

It would be better to add a new prop to override `ErrorDetailsList`.  That way the parent could provide a custom component to render the error, based on their use case.

Reply:
It would need to get propagated through `FileUpload` as well.